doc/decisions/0002-define-proxy-mechanism.md

```diff
@@ -319,34 +329,25 @@ For questions, the basic premise is to use the multiple question scheme that is
                     }
                 ]
             }]
-        }
-    },
-    "variant_choice": [
-        {
-            "variant": "example-question-proxy",
-            "when": [{
-                "id": "proxy-answer",
-                "condition": "equals",
-                "value": "proxy"
-            }]
         },
-        {
-            "variant": "example-question"
-        }
-    ]
+        "when": [{
+            "id": "proxy-answer",
+            "condition": "equals",
+            "value": "proxy"
+        }]
+    }]
 }
 ```
 
-Each variant should have the same structure as questions currently do, including the `id`. The key of each object in `variants` is a label for that variant and does not necessarily correspond to the question id.
-
-Skip conditions on questions have been moved to the top level of the block and into `variant_choice`. This enforces a seperation between the questions themselves and the logic around whether or not they should be displayed.
+Each variant should have the same structure as questions currently do. 
 
-The schema should intitially still allow `question` to be used for questions which don't require a variant form. The question should still use the same object schema as currently found within `questions`.
+The schema should initially still allow `questions` to be used for questions which don't require a variant form. This will eventually be changed to use `question` and enforce that it contains a single question.
```

Comment:
I thought we were removing `questions` as part of the breaking changes we're making in this pr?

Reply:
Yep, I just failed to update it.